client/core/core.go

```diff
@@ -900,10 +900,12 @@ func (c *Core) connectedDEX(addr string) (*dexConnection, error) {
 	return dc, nil
 }
 
-// setEpoch sets the epoch. If the passed epoch is greater than the highest
-// previously passed epoch, an epoch notification is sent to all subscribers and
-// true is returned.
-func (dc *dexConnection) setEpoch(mktID string, epochIdx uint64) bool {
+// tryUpdateEpoch updates the epoch for market with mktID to epochIdx,
+// sends an epoch notification to all subscribers
+// and returns true only
+// if the passed epoch is greater than the highest previously observed epoch.
+// Otherwise tryUpdateEpoch returns false.
```

Comment:
I know this gripe seems silly, but it's really not an burden to just setup your editor with a key combo to automatically wrap selected text.  It takes virtually no thought, it makes docs look consistent, and we all do it unless the content of the comment demands an exception.  This looks super janky right now.

Reply:
I just use `:set colorcolumn=80` with vim and do it manually though that's a little "neckbeard"ish.

Reply:
Changed formatting for this section.

I'm formatting everything manually at the moment the way that seems to be the most readable to me, but looks like it's not working as expected :)

Long-term, I should probably setup something to help me. Other comments look OK ?